skipruntime-ts/core/src/skipruntime_api.ts

```diff
@@ -1,16 +1,26 @@
 /**
- * This file contains the SKStore public API: types, interfaces, and operations that can be
- * used to specify and interact with reactive computations. See [todo: pointer to public
- * overview page] for a detailed description and introduction to the SKStore system.
+ * This file contains the Skip Runtime's public API: types, interfaces, and operations that
+ * can be used to specify and interact with reactive computations. See [todo: pointer to
+ * public overview page] for a detailed description and introduction to Skip.
  */
 
 import type { Opt, Shared, int } from "std";
 import type { Constant } from "./internals/skipruntime_impl.js";
 
+/**
+ * The `TJSON` type describes JSON-serializable values and serves as an upper bound on keys
+ * and values in the Skip Runtime, ensuring that they can be serialized and managed by the
+ * reactive computation engine.
+ */
+export type TJSON = number | boolean | string | JSONObject | (TJSON | null)[];
 export type JSONObject = { [key: string]: TJSON | null };
 
-export type TJSON = number | JSONObject | boolean | (TJSON | null)[] | string;
-
+/**
+ * A `Param` is a valid parameter to a Skip runtime mapper function: either a constant JS
+ * value or a Skip-runtime-managed value.  In either case, restricting mapper parameters to
+ * this type ensures that reactive computations can be re-evaluated as needed with consistent
```

Comment:
Not for this PR, but maybe "ensures" is too strong here, since if users try they can break it.